sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -15,7 +15,11 @@
 
 #define MAX_NO_PROPS 2
 #define MAX_HDMI_NUM 4
-#define SDW_AMP_DAI_ID 2
+#define SDW_UNUSED_DAI_ID 0
+#define SDW_JACK_IN_DAI_ID 0
```

Comment:
Sorry @bardliao, is it intentional that 'unused' and 'jack_in' have the same ID? That seems wrong.

Reply:
> Sorry @bardliao, is it intentional that 'unused' and 'jack_in' have the same ID? That seems wrong.

Yes, it was intentional . I use both 0 before using a macro. But, yes, it is better to use different number. Changed SDW_UNUSED_DAI_ID to -1 and added a check after getting dailink id now.